test/e2e/seeder/network-states/withBalanceCheckerStateless.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
ℹ️ no need to modify the storage in the balance checker contract, as this will simply call the ERC20 contract(s), so the balance for the tokens has to be modified only at the ERC20 contract